webapp/src/aws-exports.js

```diff
@@ -17,8 +17,14 @@ const awsmobile = {
         "redirectSignIn": environment.redirectSignIn,
         "redirectSignOut": environment.redirectSignOut,
         "responseType": "code"
+    },
+    "identityProviders": {
+        "GitHub": {
+            "client_id": "YOUR_GITHUB_CLIENT_ID",
+            "client_secret": "YOUR_GITHUB_CLIENT_SECRET",
```

Comment:
`YOUR_GITHUB_CLIENT_SECRET` should not be exposed in the `webapp`. This should rather be set as `env` in a proxy-lambda. This proxy lambda should be responsible to authenticate to the Github OAuth Server. The proxy lambda can be configured in the Cognito `UserPoolIdentityProvider`.